engine/src/main/java/com/datatorrent/stram/StreamingAppMasterService.java

```diff
@@ -644,6 +644,15 @@ protected void serviceStart() throws Exception
       if (SecurityUtils.isStramWebSecurityEnabled()) {
```

Comment:
I like the idea of the block moving to SecurityUtils. Better than changing the visibility of a private member of SecurityUtils to public. The function can return a Configuration object.